### ISSUE TITLE
`require-returns` requires `@return` tags for empty return statements

### ISSUE THREAD

**Issue by pelepelin** [gajus/eslint-plugin-jsdoc#158]
Using 4.1.0

For the purpose of `require-returns`, the function body that contains only return statements without expressions should be considered equivalent to a function without return statements.
The following examples should be considered valid:
```js
/**
 *
 */
function foo() {
  return;
}

/**
 *
 */
function foo() {
  if (1) {
    return;
  } else {
    return;
  }
}
```

**Comment by pelepelin** [comment]
This is the reverse of #142

**Comment by pelepelin** [comment]
Oops. A duplicate of #149 which is closed. Does #142 cover both cases?

**Comment by mattlyons0** [comment]
I am using `4.8.2` (which is supposed to have the fix for #142) and still seeing ```function foo() { return }``` say it is missing the return function

**Comment by brettz9** [comment]
Fixed by #217.
